Scripts/Runtime/Entities/Tasks/Data/UpdateJobConfig.cs

```diff
@@ -0,0 +1,50 @@
+using Anvil.Unity.DOTS.Jobs;
+using Unity.Entities;
+using Unity.Jobs;
+
+namespace Anvil.Unity.DOTS.Entities
+{
+    public class UpdateJobConfig<TData>
+        where TData : unmanaged, IProxyData
+    {
+        public delegate JobHandle ScheduleJobDelegate(JobHandle jobHandle, UpdateJobData<TData> jobData, IScheduleInfo scheduleInfo);
+        
+        private readonly IScheduleInfo m_ScheduleInfo;
```

Comment:
Should this be `ProxyDataStreamScheduleInfo` since you are controlling the type in here?

Can the delegate also use the concrete type? I think Contravariance means you can provide a method instance that would accept `IScheduleInfo` and it would still work.

Reply:
Yep, good point. Fixed.